back/engines/commercial/insights/spec/models/insights/category_assignment_spec.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Insights::CategoryAssignment do
+  describe 'validations' do
+    subject(:assignment) { build(:category_assignment) }
+
+    specify { expect(assignment).to be_valid }
+
+    it 'is not valid without a view' do
```

Comment:
I don't think it's necessary to have a view here. I think that test is actually is failing :)

Reply:
Other good catch. Leftover of a previous implementation where assignment also had references to views. But it was redundant with categories. Still have to check why the assignment is not valid though. Maybe it's the failing test actually 🤞

Reply:
Good news: the test was indeed failing, + a few other tests related to dependencies between categories/views/inputs and assignments. The test related to the assignments-input dependency is still failing in this branch for now.